packages/core/src/node/process/process.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2017 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { injectable, decorate } from 'inversify';
+import { ILogger } from '../../common/logger';
+import * as events from 'events';
+import * as child from 'child_process';
+import * as stream from 'stream';
+
+decorate(injectable(), events.EventEmitter);
+@injectable()
+export abstract class Process extends events.EventEmitter {
```

Comment:
Ok, I didn't notice the forwarding. I just like our event api better as it would be delegation instead of inheritance and would go without the event-type strings.